ui/src/ReactCustomHooks/useTranslate/useTranslate.spec.js

```diff
@@ -29,9 +30,10 @@ describe('useTranslate hook', () => {
     });
   });
   it('when provided with a bundle, locale and translation key, but the key does not exist in the loaded bundles, a sensible message is returned', () => {
-    const { result: translate } = renderHook(() =>
+    const { getResultFromHook } = renderHook(() =>
       useTranslate(mockBundle, mockLocale)
     );
+    const translate = getResultFromHook();
     const key = 'NOT_EXIST';
     const expectedString = `${key} provided, but none found for locale ${mockLocale}`;
     act(() => {
```

Comment:
How was this working before you exported act?

Reply:
The `result` never changed post 1st render. Hence the issue wasnt seen

Reply:
Sorry, I mean the use of `act` - can't see where it came from before exporting it from the utils

Reply:
Ah sorry! So there are many `act` functions - we happened to be inheriting one from base React (I believe - can recreate the issue to be sure if needed). The second in my new tests I started to drive a change in state, a number of warnings regarding the wrong `act` function started firing from RTL. So to be explicit, I am now importing the `act` from RTL's hooks, which is the same as regular RTL.